tempto-core/src/main/java/com/teradata/tempto/sql/SqlContexts.java

```diff
@@ -41,6 +42,11 @@ public static ViewContextProvider createViewAs(String selectSql)
         return new ViewContextProvider(viewName, selectSql, defaultQueryExecutor());
     }
 
+    public static ViewContextProvider createViewAs(String viewName, String selectSql, QueryExecutor queryExecutor)
+    {
+        return new ViewContextProvider(viewName, selectSql, queryExecutor);
```

Comment:
Why can't you use defaultQueryExecutor? Your commit message should ideally tell why something is being done rather than what is being done.